docs/index.md

```diff
@@ -3,16 +3,22 @@
     **This is very much a work in progress!**<br/>
     Many pages are still empty, that will hopefully change soon...
 
-# European Environment for Scientific Software Installations (EESSI)
+![EESSI logo](img/logos/EESSI_logo_horizontal.png)
 
----
+# Welcome to the EESSI project documentation!
 
-## Contents
+The **European Environment for Scientific Software Installations** (EESSI, pronounced as "easy") is a collaboration between different European partners in HPC community.
 
-* [General info](00_general.md)
-* [Overview](01_overview.md)
-* [CVMFS layer](11_cvmfs.md)
-* [Gentoo Prefix layer](12_gentoo_prefix.md)
-* [EasyBuild layer](13_easybuild.md)
-* [Project partners](90_partners.md)
-* [Contact info](99_contact.md)
+The goal of this project is to build a common scientific software stack for HPC systems and beyond, including laptops,
+personal workstations and cloud infrastructure.
+
+The concept is heavily inspired by the <a href="https://docs.computecanada.ca/wiki/Accessing_CVMFS">Compute Canada software stack</a>.
```

Comment:
Not a problem but a question: do we care about mixing markup and html formatting?

Reply:
As long as the link works, not really I guess. 

But I should have used a Markdown link here, I'll fix that in the next PR